atom/browser/api/atom_api_web_contents.cc

```diff
@@ -2065,7 +2065,7 @@ void WebContents::AddWorkSpace(mate::Arguments* args,
     args->ThrowError("path cannot be empty");
     return;
   }
-  DevToolsAddFileSystem(path);
+  DevToolsAddFileSystem(path, std::string());
```

Comment:
just an empty string?

Reply:
empty means `default` and also means `add to workspaces`.
This is electron defined usage and it didn't expect there will be more than workspaces filesystem type